test/instrumentation/modules/memcached.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+var agent = require('../../..').start({
+  serviceName: 'test',
+  secretToken: 'test',
+  captureExceptions: false,
+  metricsInterval: 0
+})
+
+var test = require('tape')
+var mockClient = require('../../_mock_http_client')
+test(function (t) {
+  resetAgent(function (data) {
+    t.equal(data.transactions.length, 1)
+    t.equal(data.spans.length, 7)
+    for (var span of data.spans) {
+      t.ok(/^memcached\.(get|set|replace|touch|delete)$/.test(span.name))
+      t.ok(/^db\.memcached\.(get|set|replace|touch|delete)$/.test(span.type))
+      t.ok(/^(get|set|replace|touch|delete) foo$/.test(span.context.db.statement))
```

Comment:
Rather than this overly permissive loop, can you check for each span in order, so we know we're actually getting all different values?

Reply:
Will do.